zgw/zaken-api/src/main/kotlin/com/ritense/zakenapi/domain/ZaakTypeLink.kt

```diff
@@ -45,10 +45,10 @@ data class ZaakTypeLink(
     @JsonProperty("id")
     val zaakTypeLinkId: ZaakTypeLinkId,
 
-    @Column(name = "document_definition_name", columnDefinition = "VARCHAR(50)", nullable = false)
+    @Column(name = "case_definition_id", columnDefinition = "VARCHAR(50)", nullable = false)
```

Comment:
When linking to a case definition you don't need the @column annotation because it's already present in the CaseDefinitionId class. The lengtrh and notblank annotation i think are also not needed.

This does require you to add the column `case_definition_key` and `case_definition_version_tag` in the database.